kong/clustering/services/sync/rpc.lua

```diff
@@ -300,8 +312,11 @@ local function do_sync()
   assert(type(kong.default_workspace) == "string")
 
   -- validate deltas
-  local ok, err = validate_deltas(deltas, wipe)
+  local ok, err, err_t = validate_deltas(deltas, wipe)
   if not ok then
+    notify_error(ns_delta.lastest_version or
```

Comment:
Could we move `v02_get_deltas_does_not_return_latest_version_field` into function notify_error?

Reply:
fixed